pages/services/confluent-zookeeper/2.5.0-4.1.3e/security/index.md

```diff
@@ -0,0 +1,299 @@
+```json
+{
+    "service": {
+        "security": {
+            "kerberos": {
+                "enabled": true,
+                "kdc": {
+                    "hostname": "<kdc host>",
+                    "port": <kdc port>
+                },
+                "primary": "<service primary default zookeeper>",
+                "realm": "<realm>",
+                "keytab_secret": "<path to keytab secret>",
+                "debug": <true|false default false>
+            }
+        }
+    }
+}
+```
+
+*Note*: It is possible to enable Kerberos after initial installation but the service may be unavailable during the transition. Additionally, your ZooKeeper clients will need to be reconfigured. For more information see the [Enabling Kerberos after deployement](#enabling-kerberos-after-deployment) section.
+
+#### Enabling Kerberos After Deployment
+
+It is possible to enable Kerberos authentication after the deployment of DC/OS {{ model.techName }}. As described in the (Rolling Upgrade)[https://cwiki.apache.org/confluence/display/ZOOKEEPER/Server-Server+mutual+authentication] section of the Apache ZooKeeper documentation, this requires multiple rolling restarts of the ZooKeeper ensemble and client connectivity may be lost at times.
+
+Assuming that DC/OS {{ model.techName }} was initially deployed with `service.security.kerberos.enabled` set to `false`, the following steps can be used to enable Kerberos for the service.
+
+Firstly -- assuming the same Kerberos settings as discussed in [Configure Kerberos Authentication](#configure-kerberos-authentication) -- create the keytab for the Kerberos principals and add this keytab to the DC/OS Secret Store as described in the [Create principals](#create-principals) and [Place Service Keytab in DC/OS Secret Store](#place-service-keytab-in-dc-os-secret-store) sections. Then create a `kerberos-toggle-step-1.json` file with the following contents:
+
+```json
+{
+    "service": {
+        "security": {
+            "kerberos": {
+                "enabled": true,
+                "kdc": {
+                    "hostname": "<kdc host>",
+                    "port": <kdc port>
+                },
+                "primary": "<service primary default zookeeper>",
+                "realm": "<realm>",
+                "keytab_secret": "<path to keytab secret>",
+                "debug": <true|false default false>,
+                "advanced": {
+                    "required_for_quorum_learner": false,
+                    "required_for_quorum_server": false,
+                    "required_for_client": false
+                }
+            }
+        }
+    }
+}
+```
+where it is important to note the `service.security.kerberos.advanced` section that is present here.
+
+Using this config file, update your DC/OS {{ model.techName }} service:
+```bash
+$ dcos kafka-zookeeper --name=<service name> update start --options=kerberos-toggle-step-1.json
```

Comment:
Should this really be issuing a kafka-zookeeper command, instead of a confluent-zookeeper command?

Reply:
nice catch! it was wrong in previous versions also
I have updated the command in old versions too.

Reply:
Thanks!